AzureSignTool/SipExtensionFactory.cs

```diff
@@ -25,6 +25,7 @@ AuthenticodeSignerAttributes attributes
             var extension = Path.GetExtension(filePath).ToLower();
             switch (extension)
             {
+                case ".msix":
```

Comment:
Do we know if this is really all it is … is it the same SIP? I have to imaging it is?

If so, there's also `.msixbundle` Not sure about the encrypted stuff (if that even works...)

Reply:
Bah that change wasn't support to land.